src/serverModel.ts

```diff
@@ -46,7 +46,10 @@ export interface IServerV2DTO {
     dp_v?: number;
     n?: string;
     et?: number;
+
+    // https://go.mparticle.com/work/SQDSDKS-5196
     ua?: Dictionary<string | string[]>;
+
```

Comment:
is this whitespace intentional?

Reply:
Yes. It's to show that the comment only applies to that specific type definition.